src/services/qAccService.ts

```diff
@@ -195,6 +202,58 @@ const getQAccDonationCap = async ({
   }
 };
 
+const getUserRemainedCapBasedOnGitcoinScore = async ({
+  projectId,
+  user,
+}: {
+  projectId: number;
+  user: User;
+}): Promise<number> => {
+  if (
+    !user.analysisScore ||
+    !user.passportScoreUpdateTimestamp ||
+    user.passportScoreUpdateTimestamp.getTime() >
+      Date.now() - GITCOIN_PASSPORT_EXPIRATION_PERIOD_MS
```

Comment:
Two points, 
1. It seems it must be to make sense
```suggestion
    user.passportScoreUpdateTimestamp.getTime() <
      Date.now() - GITCOIN_PASSPORT_EXPIRATION_PERIOD_MS
```
2. It seems we don't update passportScoreUpdateTimestamp anymore in  `updateUserGitcoinAnalysisScore`. Please revise the way it's used.